src/fsharp/xlf/FSComp.txt.it.xlf

```diff
@@ -152,6 +152,11 @@
         <target state="translated">gestione pacchetti</target>
         <note />
       </trans-unit>
+      <trans-unit id="featurePrintfBinaryFormat">
+        <source>printf format for binary integers</source>
+        <target state="new">printf format for binary integers</target>
+        <note />
+      </trans-unit>
```

Comment:
I find these update automatically when I do a local build, so not quite sure why you had to do it manually.. :-)

Reply:
I did everything in this PR through the web editor because my weak computer cannot handle large solutions, and when these files are out-of-sync tests will fail